docs/services/slam/cartographer/_index.md

```diff
@@ -568,68 +407,77 @@ If you move your machine, it will appear to be moving in a trajectory from the m
 <!-- prettier-ignore -->
 | Name | Data Type | Required? | Description |
 | ---- | --------- | --------- | ----------- |
-| `use_cloud_slam` | boolean | **Required** | If `true`, the Cartographer algorithm will execute in the cloud rather than locally on your machine. |
+| `use_cloud_slam` | boolean | **Required** | If `true`, the Cartographer algorithm will disable mapping on the machine. This feature should only be used when trying to use [cloudslam](../cloudslam/). |
 | `camera` | obj | **Required** | An object of the form `{ "name": <string>, "data_frequency_hz": <int> }` where `name` is the name of the LiDAR camera component to use as input and `data_frequency_hz` is the rate at which to capture (in "Create new map" or "Update existing map" modes) or poll (in "Localize only" mode) data from that camera component. |
 | `movement_sensor` | obj | Optional | An object of the form `{ "name": <string>, "data_frequency_hz": <int> }` where `name` is the name of the IMU movement sensor (that is, a movement sensor that supports the `GetAngularVelocity` and `GetLinearAcceleration` API methods) to use as additional input and `data_frequency_hz` is the rate at which to capture (in "Create new map" or "Update existing map" modes) or poll (in "Localize only" mode) data from that movement sensor component. |
 | `enable_mapping` | boolean | Optional | If `true`, Cartographer will build the map in addition to doing localization. <ul> Default: `true` </ul> |
-| `existing_map` | string | Optional | The alias of the package containing the existing map to build on (in "Update existing map" mode) or localize on (in "Localize only" mode). |
+| `existing_map` | string | Optional | The alias of the package containing the existing map to build on (in "Update existing map" mode) or localize on (in "Localize only" mode). Can also point to a locally saved `.pbstream` file. |
```

Comment:
```suggestion
| `existing_map` | string | Optional | The alias of the package containing the existing map to build on (in "Update existing map" mode) or localize on (in "Localize only" mode). Can also point to a locally saved `.pbstream` file if you have completed creating or updating your map locally on the robot  cloud connectivity. |
```

Is this what you're getting at?

Reply:
I don't know what "robot cloud connectivity" has to do with anything here. The intent is just to say `if you have a local .pbstream file, you can use that too`

Reply:
Yeah but when will they want to use the local pbstream is unclear, even to me.

Reply:
they want to use a local pbstream when they don't want to use cloudslam. i.e. they don't want to spend money/touch the $5 budget. I am intentionally not calling that detail out, but I can if you think we should